src/main/java/com/elastic/support/rest/RestEntryBuilder.java

```diff
@@ -0,0 +1,62 @@
+package com.elastic.support.rest;
+
+import com.elastic.support.util.SystemProperties;
+import com.sun.org.apache.xpath.internal.operations.Bool;
+import com.vdurmont.semver4j.Semver;
+import jdk.internal.jshell.tool.resources.version;
+import org.apache.commons.lang3.ObjectUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import javax.swing.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class RestEntryBuilder {
```

Comment:
Nitpick: This is more of a `Factory` than a `Builder`. It also doesn't seem to be used, but I'm guessing that's for the next PR?

Reply:
True, it changed substantially at one point and I never renamed it. Next PR will have the implementation. At some point I'll probably rename it but I had other higher priorities.